Src/DependencyCollector/DependencyCollector/Microsoft.ApplicationInsights.DependencyCollector.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="CopyApplicationInsightsConfigToOutputDirectory" AfterTargets="CopyFilesToOutputDirectory" >
```

Comment:
Is `CopyFilesToOutputDirectory` executes for console apps?

Reply:
I can test specifically with console apps. Would we need to take any action if `CopyFilesToOutputDirectory` is not executed?

Reply:
I confirmed that this is executed for Console apps.

I would argue though that we would be unaffected by any app type that does not copy files to an output directory. :)